### ISSUE TITLE
Try to improve search results

### ISSUE THREAD

**Issue by PTKay** [HedgeDocs/HedgeDocs.github.io#13]
The current way we show search results is not ideal. Here's an example:

<img src='https://user-images.githubusercontent.com/20132880/200284676-b35045a7-22f3-47d7-8daa-ff530c9b6e6a.png' height='750'>

The results we get aren't very informative, and instead consist on simply showing the title and part of the page's text, if that even decides to show up. We should try to make the search results show the page's description in the search results, as a way to get a better overview of what the page is about.

Additional suggestions to the search results are also welcome, and should be posted on this issue as comments :)

## TLDR
Change search results from
> Page Title
Page Text

to

> Page Title
Page Description (metadata)
